### PR TITLE
fix(renovate): provide complete packageName if git-refs is used as datasource

### PR DESCRIPTION
**Pull request by williamboman**

### CODE REVIEW

**Comment by williambotman** [comment]
@mason-org/triage `notify(Reason::NewPullRequest)`

<sup>This comment ensures a notification is delivered to maintainers.</sup>

<sup>` 🤖 This is an automated comment. `  [` 📖 Source code `](https://github.com/williamboman/botman)</sup>

**Comment by williamboman** [comment]
@williambotman /fixup

File: renovate.json5 [hunk 1]
```diff
@@ -20,6 +20,7 @@
             ],
             datasourceTemplate: "{{#if datasource}}{{{datasource}}}{{else}}github-releases{{/if}}",
             versioningTemplate: "{{#if versioning}}{{{versioning}}}{{else}}semver{{/if}}",
+            "packageNameTemplate": "{{#if datasource == 'git-refs'}}https://github.com/{{packageName}}{{else}}{{packageName}}{{/if}}",
```
Comment (williamboman) [inline renovate.json5:23]:
```suggestion
            "packageNameTemplate": "{{#if (containsString datasource 'git-refs')}https://github.com/{{packageName}}{{else}}{{packageName}}{{/if}}",
```

File: renovate.json5 [hunk 2]
```diff
@@ -20,6 +20,7 @@
             ],
             datasourceTemplate: "{{#if datasource}}{{{datasource}}}{{else}}github-releases{{/if}}",
             versioningTemplate: "{{#if versioning}}{{{versioning}}}{{else}}semver{{/if}}",
+            "packageNameTemplate": "{{#if (containsString datasource 'git-refs')}https://github.com/{{packageName}}{{else}}{{packageName}}{{/if}}",
```
Comment (williamboman) [inline renovate.json5:23]:
```suggestion
            packageNameTemplate: "{{#if (containsString datasource 'git-refs')}https://github.com/{{packageName}}{{else}}{{packageName}}{{/if}}",
```

File: renovate.json5 [hunk 3]
```diff
@@ -20,6 +20,7 @@
             ],
             datasourceTemplate: "{{#if datasource}}{{{datasource}}}{{else}}github-releases{{/if}}",
             versioningTemplate: "{{#if versioning}}{{{versioning}}}{{else}}semver{{/if}}",
+            packageNameTemplate: "{{#if (containsString datasource 'git-refs')}https://github.com/{{packageName}}{{else}}{{packageName}}{{/if}}",
```
Comment (williamboman) [inline renovate.json5:23]:
```suggestion
            packageNameTemplate: "{{#if (containsString datasource 'git-refs')}https://github.com/{{{packageName}}}{{else}}{{{packageName}}}{{/if}}",
```

File: renovate.json5 [hunk 4]
```diff
@@ -20,6 +20,7 @@
             ],
             datasourceTemplate: "{{#if datasource}}{{{datasource}}}{{else}}github-releases{{/if}}",
             versioningTemplate: "{{#if versioning}}{{{versioning}}}{{else}}semver{{/if}}",
+            packageNameTemplate: "{{#if (containsString datasource 'git-refs')}https://github.com/{{{packageName}}}{{else}}{{{packageName}}}{{/if}}",
```
Comment (williamboman) [inline renovate.json5:23]:
```suggestion
            packageNameTemplate: "{{#if (containsString datasource 'git-refs')}}https://github.com/{{{packageName}}}{{else}}{{{packageName}}}{{/if}}",
```

**Pull request merged**